client/src/main/java/com/influxdb/client/write/events/WriteRetriableErrorEvent.java

```diff
@@ -67,7 +67,7 @@ public Long getRetryInterval() {
 
     @Override
     public void logEvent() {
-        String msg = "The retriable error occurred during writing of data. Retry in: {0} [ms]";
-        LOG.log(Level.WARNING, msg, retryInterval);
+        String msg = "The retriable error occurred during writing of data. Reason: '{0}'. Retry in: {1}ms.";
```

Comment:
May be, better to use seconds instead of miliseconds ?

Reply:
Done